package.json

```diff
@@ -4,6 +4,11 @@
   "description": "Cumulus Framework for ingesting and processing NASA Earth data streams",
   "scripts": {
     "audit": "audit-ci --config ./audit-ci.json && lerna exec -- audit-ci --config $(pwd)/audit-ci.json",
+    "deploy": "npm run deploy-cumulus",
+    "deploy-all": "npm run deploy-data && npm run deploy-cumulus",
+    "predeploy-cumulus": "npm install && npm run bootstrap",
+    "deploy-cumulus": "cd example/cumulus-tf && terraform init && terraform apply -auto-approve",
```

Comment:
I personally don't love having a script that does `terraform apply -auto-approve` in the repo, because it's easy to deploy changes you didn't intend to, but think I'll be outvoted on that one

Reply:
I'm open to suggestions for improvement. I could drop the `-auto-approve` flag, but support adding it to the command line. For example `npm run deploy -- -auto-approve` (which would actually allow you to add _any_ valid `terraform` option).  What do you think about that?

Reply:
There are other much more important changes in this PR so let's just not worry about this one. It's just my opinion anyway